packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -533,6 +533,13 @@ class _MonthPickerState extends State<MonthPicker> {
     _dayPickerController = new PageController(initialPage: monthPage);
     _handleMonthPageChanged(monthPage);
     _updateCurrentDate();
+
+    // Setup the fade animation for chevrons
+    _chevronOpacityController = new AnimationController(
+      duration: const Duration(milliseconds: 500), vsync: this
+    );
+    final curve = new CurvedAnimation(parent: _chevronOpacityController, curve: Curves.easeInOut);
```

Comment:
A type is needed here, `Animation<double>`, and maybe curve is the wrong name for the variable since it's not a Curve. 

Sorry to nit pick. You could avoid the whole issue by just writing:
```
_chevronOpacityAnimation = new Tween(begin: 1.0, end: 0.5).animate(
  new CurvedAnimation(
    parent: _chevronOpacityController,
    curve: Curves.easeInOut,
  ),
);

```

Reply:
Nitpick away as it leads to cleaner code! Will revise to clean up.